src/header.rs

```diff
@@ -91,6 +91,7 @@ pub struct NiftiHeader {
     /// Grid spacings
     pub pixdim: [f32; 8],
     /// Offset into .nii file to reach the volume
+    /// Note: the highly unusual choice of f32 is intentional and due to trying to achieve header backwards compatibility to ANALYZE 7.5
```

Comment:
Adding a newline so that the small description does not get cluttered with the note.
```suggestion
    /// Offset into .nii file to reach the volume.
    ///
    /// Note: the highly unusual choice of f32 is intentional
    /// and due to trying to achieve header backwards compatibility to ANALYZE 7.5
```